.golangci.yml

```diff
@@ -137,6 +137,9 @@ issues:
         - prealloc
     - path: ^test/
       text: exported (.+) should have comment( \(or a comment on this block\))? or be unexported
+    - linters:
+        - staticcheck
+      text: "SA1019:"
```

Comment:
is it possible to make this exclusion more targetted to make sure it only applies to the SDK functions and not everything in the codebase?

Reply:
I just enriched the text to exclude by additionally referring to the prefix of SDKv1's pkg (found from https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-azure/2899/pull-cluster-api-provider-azure-test/1599842175804248064/build-log.txt). Hope it works as intended. PTAL. Thanks!